repos/system_upgrade/el7toel8/actors/setuptargetrepos/actor.py

```diff
@@ -80,7 +80,7 @@ def process(self):
         for used_repos in self.consume(UsedRepositories):
             for used_repo in used_repos.repositories:
                 used[used_repo.repository] = used_repo.packages
-                for repo in repo_file.data:
+                for repo in used_repo.data:
```

Comment:
My first attempt to fixing it was exactly like yours but I'm not sure it will work this way :(
`repo_file` is of type [RepositoryFile](https://github.com/oamg/leapp-repository/blob/master/repos/system_upgrade/el7toel8/models/repositoriesfacts.py#L17) and has a data field (a list of RepositoryData objects).
While `used_repo` is of type [UsedRepository](https://github.com/oamg/leapp-repository/blob/master/repos/system_upgrade/el7toel8/models/usedrepositories.py#L5) which doesn't have data field at all.

Reply:
```suggestion
```